### ISSUE TITLE
f.cesm3.cam6_3_092.FCMTHIST.ne30.001

### ISSUE THREAD

**Issue by tilmes** [NCAR/amwg_dev#215]
### Description of the run

Development version cam6_3_092 test simulation for the L93 model with Chemistry.
This is to test the current implementation with MAM5 and physcis reordering using cam_dev. 
The idea is to do some initial dust and lightning tuning and check if the chemistry and dynamics is reasonable (despite upcoming changes in clouds etc).  Start in 1995, to make it easy to compare to observations.
Source Code:  /glade/work/fvitt/camdev/cam6_3_092

### Suffix in the casename

f.cesm3.cam6_3_092.FCMTHIST.ne30.001

### Namelist modifications

/glade/p/cesmdata/cseg/runs/cesm2_0/f.cesm3.cam6_3_092.FCMTHIST.ne30.001/user_nl_cam

### Sandbox

 /glade/work/fvitt/camdev/cam6_3_092

### Contact info

@tilmes

**Comment by adamrher** [comment]
> the 26 series runs also don't show this

The 26 series uses the re-ordering. Since you're comparing to a f09 run from 2021, many things could explain the differences, no? The change in dycore and vertical grid to mention a couple ...

I'd like to try to get all these microphysics issues into git issues. They are getting hard to keep track of. Only these two issues https://github.com/ESCOMP/CAM/issues/746 https://github.com/ESCOMP/CAM/issues/751 are currently up. What are we missing?

**Comment by tilmes** [comment]
@adamrher First, I compare 1995-1997 to 2001-2003, that is very similar, and ozone is not changing that much in that time.
My understanding was that you just added the reordering into cam_dev in the development code, and you just emailed me earlier, that if I run without cam_dev this would not be include. Is that correct? I am just running a case without cam_dev (so with cam6 physics) that should show if it is the cause, correct?

**Comment by tilmes** [comment]
@adamrher the new case I am running is here:
/glade/p/cesmdata/cseg/runs/cesm2_0/f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001/user_nl_cam

**Comment by adamrher** [comment]
Since cam_dev turns on (a) reordering, (b) ZM2, and (c) PUMASv1, it's going to be challenging to isolate these. You can control ZM2 with ```zmconv_parcel_pbl``` (set to true for ZM2). The other two are not very practical to revert.

I still suspect it's because you're comparing to a different dycore and vertical grid. We've shown w/ the dry bias that nudging can't correct for differences due to dycore and vertical grid.

**Comment by tilmes** [comment]
@adamrher would this do it:  <entry id="CAM_CONFIG_OPTS" value="-phys cam6 -chem trop_strat_mam5_vbs -nlev 93"> (this is what I am running now).
Second, I have done the L58 run, see https://github.com/NCAR/amwg_dev/issues/213
The problem with ozone is not there:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3_cam058_mom_e.FCHIST.ne30_L58.26c_non-orogw_off.topofix.001.2001_2010-FCnudged_f09.mam.mar27.2000_2021.001.2001_2010/cset3/cset3_regions_taylor.png

**Comment by adamrher** [comment]
got it. thanks for providing that 26 series diags. It is a different vertical grid (L58 vs L93), but it does some more plausible to me that it could be the physics after seeing those plots.

**Comment by tilmes** [comment]
If the run that I am doing now does not show the problem, that it must be one of the three things:
(a) reordering, (b) ZM2, and (c) PUMASv1  , correct? We can dig into it after doing a couple of years.

**Comment by tilmes** [comment]
I can also run L58, but different GW will not change surface ozone.  Can you let me know what settings to set if you think, I should run something different?

**Comment by adamrher** [comment]
I rec'd turning on ZM2 in your planned -phys cam6 run.

**Comment by tilmes** [comment]
what do I set the CAM config settings to?

**Comment by adamrher** [comment]
-phys cam6 and then set zmconv_parcel_pbl=.true. in user_nl_cam

**Comment by tilmes** [comment]
Thanks, I will do a second test run with those settings, so we can compare both.

**Comment by tilmes** [comment]
@adamrher Here are the diagnostics, just running one year with the cam6 setting (not the mg2 yet):
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001.1995_1995-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/
Ozone is already showing an increase, so it is likely to get worse with time:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001.1995_1995-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/cset3/cset3_regions_taylor.png

I will post the results of the other run, when it is done.

**Comment by adamrher** [comment]
@tilmes do you have those diagnostic tables you made when we did the emitb4turb vs. cam_dev experiments a while back? What did emitb4turb do to ozone?

**Comment by adamrher** [comment]
ok. that may be the culprit if the ozone magnitudes are similar to our emitb4turb experiment. This went on the trunk in cam6_3_082, and is invoked with cam_dev.

**Comment by tilmes** [comment]
Correct, I just also did the diagnostics for the -phys cam6 with mg2:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.mg2.FCMTHIST.ne30.001.1995_1995-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/
and it also has this problem.
So, I am thinking that, to test this, is there a way we could revert the emitb4turb as the first test, and then I can compare to my  first sandbox case.  Is there code you could provide to the sandbox I used that would revert just this one thing?

Second, we need to go back and understand what is happening. I saw before that ozone changed, but wanted to wait to see if this is still in the fully coupled L93 model. My suspicion is that potentially too much aerosol is mixed up, and not removed sufficiently. It may be that  various schemes of dry and wet removal etc. are adjusted to the old way, and when we change it everything is getting messed up. The question will be what we do with this. I will also discuss this with our chemistry group.

**Comment by tilmes** [comment]
Btw, this change will strong impact the clouds, so that may be also a problem. I see a very strong SWCF in the first simulation I did. Tuning the clouds makes only sense if the aerosols are reasonable, and I think, they may not be, but we have to test.

**Comment by adamrher** [comment]
sorry I'm getting confused. Is it true that your run with -phys cam_dev and this latest run with -phys cam6, are giving large ozone values?

**Comment by tilmes** [comment]
yes, they are all showing larger ozone values, the last two with -phys cam6 show smaller changes, but only because I only run 1 year. To best check it would be to do a longer run (5years or so) with the reverted emitb4turb, that would really show if this change has the impact that we are seeing. Then we can discuss what to do next?

**Comment by adamrher** [comment]
To be clear, emitb4turb is not on in the -phys cam6 simulations.

**Comment by tilmes** [comment]
Oh, well I misunderstood. In that case, I will continue my mg2 run for a few more years and let you know what the differences are, does that sound like a good plan.

**Comment by lkemmons** [comment]
Note: I am 'lkemmons' - no idea who 'Emmons' is.

**Comment by tilmes** [comment]
I performed 2 more simulations
1. This simulation uses cam6 physics without any  other changes. The diagnostics are here:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001.1996_1998-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/
Ozone diagrams are here (red is the new run)
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001.1996_1998-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/cset3/cset3_regions_taylor.png

2. The second simulation uses cam6 physics and zm2: zmconv_parcel_pbl=.true.   The diagnostics are here:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.mg2.FCMTHIST.ne30.001.1996_1998-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/
Ozone comparisons are here:
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.cam_no_dev.mg2.FCMTHIST.ne30.001.1996_1998-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/cset3/cset3_regions_taylor.png

I have to look at this more but at least ozone loss better in the second simulation. There are some problems in the first test simulation in ozone that have to be looked at, but it is somewhat better, than the original L93 simulation.

**Comment by tilmes** [comment]
The case of these simulations above are in:
1. /glade/p/cesmdata/cseg/runs/cesm2_0/f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001/
2. /glade/p/cesmdata/cseg/runs/cesm2_0/f.cesm3.cam6_3_092.cam_no_dev.mg2.FCMTHIST.ne30.001/
The output is here:
/glade/scratch/tilmes/archive
f.cesm3.cam6_3_092.cam_no_dev.FCMTHIST.ne30.001
and f.cesm3.cam6_3_092.cam_no_dev.mg2.FCMTHIST.ne30.001

**Comment by tilmes** [comment]
Here are the diagnostics from a run that includes emissions before diffusion (and not emisb4turb):
https://acomstaff.acom.ucar.edu/tilmes/amwg/cam7/f.cesm3.cam6_3_092.emitb4diff.FCMTHIST.ne30.001.1996_1998-FCnudged_f09.mam.mar27.2000_2021.001.2001_2004/

**Comment by adamrher** [comment]
I think some of the confusion is that you are calling this emissions before diffusion when it is not. "diffusion" is not actively diffusing anything in CAM6; It's disabled. All the actual diffusion occurs in the CLUBB/ice nuc/microphys box.

**Comment by adamrher** [comment]
I just deleted a comment that makes no sense.

**Comment by tilmes** [comment]
here is a table with budgets between emisb4diff compared to emisb4turb (cam_dev)
<img width="470" alt="image" src="https://user-images.githubusercontent.com/22222754/222569350-70c6c8ef-d4dc-4652-9521-1a3b22d7a8ca.png">
